src/internal/kopia/conn.go

```diff
@@ -90,13 +90,22 @@ func (w *conn) Initialize(ctx context.Context) error {
 		return clues.Wrap(err, "initialzing repo").WithClues(ctx)
 	}
 
-	return w.commonConnect(
+	err = w.commonConnect(
 		ctx,
 		cfg.KopiaCfgDir,
 		bst,
 		cfg.CorsoPassphrase,
 		defaultCompressor,
 	)
+	if err != nil {
+		return err
+	}
+
+	if err := w.setDefaultConfigValues(ctx); err != nil {
+		return clues.Stack(err).WithClues(ctx)
```

Comment:
Since you passed in ctx, you can assume that the called function sets `WithClues(ctx)` for you.
```suggestion
		return err
```